src/rpc/blockchain.cpp

```diff
@@ -383,7 +383,7 @@ void entryToJSON(UniValue &info, const CTxMemPoolEntry &e)
 {
     AssertLockHeld(mempool.cs);
 
-    info.pushKV("size", (int)e.GetTxSize());
+    info.pushKV("vsize", (int)e.GetTxSize());
```

Comment:
I imagine this key-value pair is heavily relied upon. I'd prefer if we included both keys "size" and "vsize" with the same value for at least one release.

Reply:
Agree with @MarcoFalke, this is a breaking change.

If you feel that field `size` should be removed then you must deprecate it first, and wait for the next release to actually remove it. Not sure it's worth it as the description is clear.